src/fragments/lib/xr/getting-started.mdx

```diff
@@ -1,3 +1,10 @@
+<Callout warning>
```

Comment:
Why are there XR changes in this PR?

Reply:
For some reason, it brought my old commit here. but I removed it